### ISSUE TITLE
error in PipelineClient("example.com:"): strconv.Atoi: parsing

### ISSUE THREAD

**Issue by c33s** [GlenTiki/autocannon-go#1]
os: windows

```
10 connections with 1 pipelining factor.
⠋ 2021/02/18 22:28:11 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
R2021/02/18 22:28:11 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
u2021/02/18 22:28:11 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
n2021/02/18 22:28:11 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
n2021/02/18 22:28:11 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
i2021/02/18 22:28:11 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
n2021/02/18 22:28:11 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
g2021/02/18 22:28:11 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
 2021/02/18 22:28:11 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
A2021/02/18 22:28:11 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
⠦ Running Autocannon... 2021/02/18 22:28:21 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
2021/02/18 22:28:21 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
2021/02/18 22:28:21 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
2021/02/18 22:28:21 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
2021/02/18 22:28:21 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
2021/02/18 22:28:21 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
2021/02/18 22:28:21 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
2021/02/18 22:28:21 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
 2021/02/18 22:28:21 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
 2021/02/18 22:28:21 error in PipelineClient("example.com:"): strconv.Atoi: parsing "": invalid syntax
```

**Comment by GlenTiki** [comment]
hey @c33s, Could you share the actual command you ran this with? Thanks.

**Comment by dansku** [comment]
I got the same error with 
```
./autocannon --pipelining=10 --uri=https://google.com
```

**Comment by surjit** [comment]
I also got same error 

Ubuntu 20.04 LTS
go version go1.18 linux/amd64

go run autocannon.go --pipelining=10 --uri=https://google.com
2022/04/14 09:42:05 error in PipelineClient("google.com:"): strconv.Atoi: parsing "": invalid syntax

**Comment by GlenTiki** [comment]
Fixed by #2
